### ISSUE TITLE
annotate does not find databases

### ISSUE THREAD

**Issue by AnotherSimon** [nextgenusfs/funannotate#142]
Hi Jon,

I tried adding in results from remote annotation jobs by running the command:
`funannotate annotate --input My_Bug \`
`  --antismash ./My_Bug/annotate_misc/antiSMASH.results.gbk \`
`  --iprscan ./My_Bug/annotate_misc/iprscan.xml \`
`  --phobius ./My_Bug/annotate_misc/phobius.results.txt \`
`  --cpus 24`

The process fails with this message to StdOut:
> [92m[05:43:01 PM][0m: OS: linux2, 24 cores, ~ 66 GB RAM. Python: 2.7.9
[92m[05:43:07 PM][0m: Running funannotate v1.1.1
[92m[05:43:09 PM][0m: Database files not found in /path/to/funannotate_db, run funannotate database and/or funannotate setup 

A call to `funannotate database` shows that everything is installed though:
> Funannotate Databases currently installed:
Database          Type        Version      Date         Num_Records   Md5checksum                       
pfam              hmmer3      31.0         2017-02      16712         3e47eec766a99b599cb24f28c4d393f8  
gene2product      text        1.4          2018-01-13   24951         ed8202d7ad02ed557aafdbfdb35fffb8  
interpro          xml         66.0         2017-11-23   32568         c230e27471cad4a352cbe45beada0c52  
dbCAN             hmmer3      6.0          2017-09-12   585           3cb06f6f93c72a56c9fa12a6294b41d5  
busco_outgroups   outgroups   1.0          2018-02-02   8             6795b1d4545850a4226829c7ae8ef058  
merops            diamond     12.0         2017-10-04   4968          d923f0177c6d27c3d2886c705347adc0  
mibig             diamond     1.3          2018-02-02   24085         84b3cd16e0b3b074e4b7ee18c6aa31fd  
uniprot           diamond     2018_01      2018-01-31   556568        a5b17a3129c5d8b4bed6ef1ada59a3f0  
go                text        2018-01-30   2018-01-30   47132         c7debe524c4d7d97b70aa97221c2455c  
repeats           diamond     1.0          2018-02-02   11950         4e8cafc3eea47ec7ba505bb1e3465d21  

I can't seem to find any other log files to share that might indicate where the process is braking down.

**Comment by nextgenusfs** [comment]
The method it uses to find the database folder is the same in all of the scripts, i.e. if you pass `-d, --database` option it looks in that folder, if `-d` is not passed then it tries to load from `$FUNANNOTATE_DB` environmental variable. This error suggests that it can't find one of the following files in your funannotate DB folder:
```
Pfam-A.hmm.h3p
dbCAN.hmm.h3p
merops.dmnd
uniprot.dmnd
funannotate-db-info.txt
```
It also should be printing the location of the folder at then end of that error line, since it is not, seems that maybe something is wrong with the environmental variable.

**Comment by AnotherSimon** [comment]
`$FUNANNOTATE_DB` was defined in environment variables but for some odd reason the file `Pfam-A.hmm.h3p` was missing. Pfam related files present were:

> Pfam-A.clans.tsv
Pfam-A.hmm
Pfam.version

I forced the reinstall of pfam and now the Pfam-A.hmm.h3[fimp] files were generated. Guess something went wrong when I first ran setup but perhaps `funannotate database` should have warned me that Pfam was not usable instead of spitting out a version number?
